lib/bike_brigade_web/endpoint.ex

```diff
@@ -1,6 +1,10 @@
 defmodule BikeBrigadeWeb.Endpoint do
   use Phoenix.Endpoint, otp_app: :bike_brigade
 
+  if sandbox = Application.get_env(:bike_brigade, :sandbox) do
```

Comment:
@drteeth can you explain how this works? I'm curious why we didn't have it before (i guess we didnt exercise the endpoint in the LV tests?)